Instanssi/main2019/static/main2019/js/radio.js

```diff
@@ -12,8 +12,9 @@ $(function() {
 
     function parseCal(params) {
         var oldDay = '';
-        var dateNow = new Date().toISOString();
-
+        var tzoffset = (new Date()).getTimezoneOffset() * 60000;
+        var dateNow = (new Date(Date.now() - tzoffset)).toISOString();
```

Comment:
I guess the issue being worked around here is that the dates are being compared as strings. `new Date().toISOString()` returns the time formatted as a UTC datetime, while the Google Calendar's timestamps are in +0200 (at least in my browser), leaving the comparison off by 2 hours.

I think you could avoid this timezone stuff by just grabbing the current UTC timestamp here with
```javascript
var dateNow = new Date().getTime();
```
And then checking if it falls within the event's parsed `start.dateTime` and `end.dateTime` in the `$.each` loop like
```javascript
if (dateNow >= startD.getTime() && dateNow < endD.getTime()) {
```

Reply:
Dates can be compared as strings but you are right, unix time is the better way.
New pull request coming soon.